src/renderer/store/modules/Note.js

```diff
@@ -53,16 +49,30 @@ const actions = {
         store.commit('SELECT_LOADING', true);
         store.commit('LOAD_NOTES', []);
 
-        octokit.authenticate({
-          type: 'token',
-          token: store.rootState.Settings.settings.githubPersonalAccessToken,
-        });
+        if (store.rootState.Settings.settings.githubEnterpriseUrl) {
+          octokit = Octokit({
+            baseUrl: store.rootState.Settings.settings.githubEnterpriseUrl,
+            userAgent: 'code-notes v1.2.3',
```

Comment:
Do you think that we can get the "v1.2.3" version from our `package.json` file?

Reply:
Yes, that is a valid point. I thought the user agent was something related to octokit/rest internal api, but upon reading github api docs, yes this should be code-notes + version, will fix it in both places.